frontend/apps/marketing/src/components/video/videoContentfulDefinition.ts

```diff
@@ -0,0 +1,49 @@
+// Creates a definition for the Video component to be used in Contentful Studio
+import {ComponentDefinition} from '@contentful/experiences-sdk-react';
+
+export const VideoContentfulComponentDefinition: ComponentDefinition = {
```

Comment:
Just a nit, this is more of a YouTube video component with fallback rather than a general Video component.

Reply:
We can rename it later on as the use cases come up so this sounds good

Reply:
Yeah, I think we can dive into this later if needed. In general I would keep considering this `Video` component is a general purpose `Video` component(with possible updates in the future) and then, if needed, add separate `YoutubeVideo` component later that might be a shortcut for using `Video` component